spec/models/theme_field_spec.rb

```diff
@@ -612,7 +612,7 @@ def create_yaml_field(value)
 
   describe "SVG sprite theme fields" do
     let :svg_content do
-      "<svg></svg>"
+      "<svg><symbol id='test'></symbol></svg>"
```

Comment:
No, I changed the test so that it's using `custom_svgs`, but that gives you symbols rather than svgs, so I needed a symbol to exist to see the difference.